cargo/lib/dependabot/cargo/update_checker/version_resolver.rb

```diff
@@ -295,7 +304,13 @@ def resolvability_error?(message)
           return true if message.match?(/feature `[^\`]+` is required/)
           return true if message.include?("unexpected end of input while parsing major version number")
 
-          !original_requirements_resolvable?
+          original_requirements_resolvable = original_requirements_resolvable?
+
+          if original_requirements_resolvable == true
+            false
+          elsif original_requirements_resolvable == false
+            true
+          end
```

Comment:
It's still early here, trying to understand this change. It looks the same as the code it replaces?

Reply:
The previous code returns always either `true` or `false`. The new code returns either `true`, `false`, or `nil` if we couldn't find a definitive hint about a resolvability issue.

Reply:
Satisfying RuboCop requests made it quite unreadable, I'll add a comment.

Reply:
Not a big deal, I was trying to figure out the significance of the change.

Reply:
Have a look at e1c8dceb565d02129b44b571ee46cb5814ed066f, it should be easier to read!